Can not found libfaiss.{a,so} after built

```
sudo make install
/bin/mkdir -p /usr/local/lib /usr/local/include/faiss
cp libfaiss.{a,so} /usr/local/lib
cp: cannot stat 'libfaiss.{a,so}': No such file or directory
Makefile:57: recipe for target 'install' failed
make: *** [install] Error 1

```

--- a/Makefile
+++ b/Makefile
@@ -19,11 +19,9 @@ GPU_SRC     = $(GPU_CPPSRC) $(GPU_CUSRC)
 GPU_CPPOBJ  = $(GPU_CPPSRC:.cpp=.o)
 GPU_CUOBJ   = $(GPU_CUSRC:.cu=.o)
 GPU_OBJ     = $(GPU_CPPOBJ) $(GPU_CUOBJ)
-GPU_INSTALLDIRS = $(DESTDIR)$(includedir)/faiss/gpu/{impl,utils}
 
 ifneq ($(strip $(NVCC)),)
 	OBJ         += $(GPU_OBJ)
-	INSTALLDIRS += $(GPU_INSTALLDIRS)
 	HEADERS     += $(GPU_HEADERS)
 endif
 
@@ -54,14 +52,15 @@ clean:
 # Installing
 
 install: libfaiss.a libfaiss.$(SHAREDEXT) installdirs
-	cp libfaiss.{a,$(SHAREDEXT)} $(DESTDIR)$(libdir)
+	cp libfaiss.a libfaiss.$(SHAREDEXT) $(DESTDIR)$(libdir)
 	tar cf - $(HEADERS) | tar xf - -C $(DESTDIR)$(includedir)/faiss/
 
 installdirs:
 	$(MKDIR_P) $(INSTALLDIRS)
 
 uninstall:
-	rm -f $(DESTDIR)$(libdir)/libfaiss.{a,$(SHAREDEXT)}
+	rm -f $(DESTDIR)$(libdir)/libfaiss.a \
+	      $(DESTDIR)$(libdir)/libfaiss.$(SHAREDEXT)
 	rm -rf $(DESTDIR)$(includedir)/faiss
 
 

